pyttb/tensor.py

```diff
@@ -282,58 +352,64 @@ def contract(self, i: int, j: int) -> Union[np.ndarray, tensor]:
 
     def double(self) -> np.ndarray:
         """
-        Convert tensor to an array of doubles
+        Convert tensor to an :array of doubles.
 
         Returns
         -------
-        Copy of tensor data
+        Copy of tensor data.
 
-        Example
-        -------
-        >>> X = ttb.tensor(np.ones((2,2)))
-        >>> X.double()
+        Examples
+        --------
+        >>> T = ttb.tensor(np.ones((2,2)))
+        >>> T.double()
         array([[1., 1.],
                [1., 1.]])
         """
         return self.data.astype(np.float_).copy()
 
     def exp(self) -> tensor:
         """
-        Exponential of the elements of tensor
+        Exponential of the elements of tensor.
 
         Returns
         -------
-        Copy of tensor data element-wise raised to exponential
+        Copy of tensor data wtih the exponential function applied to data
+        element-wise.
```

Comment:
I think for the multiline returns it needs to be indendented to render correctly https://pyttb--216.org.readthedocs.build/en/216/tensor.html#pyttb.tensor.exp

Reply:
This wasn't actually resolved but I just pushed a commit that should have fixed the formatting.